src/Nethermind/Nethermind.Specs/GnosisForks/18_PragueGnosis.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Threading;
+using Nethermind.Core.Specs;
+
+namespace Nethermind.Specs.GnosisForks;
+
+public class PragueGnosis : Forks.Prague
```

Comment:
Do we need `PragueLondon` to enable `1559Collector` and `FeeCollector` address? (Then more useful is to use `OverridableReleaseSpec` than just inheritance)

Reply:
I'm not sure what `PragueLondon` is? In `OverridableReleaseSpec` it says it's for testing purposes only

Reply:
sorry `LondonGnosis`